examples/core/transform-feedback-instanced/app.js

```diff
@@ -216,84 +210,44 @@ const animationLoop = new AnimationLoop({
     const positionBuffer = new Buffer(gl, {
       data: trianglePositions,
       size: 2,
-      type: gl.FLOAT,
-      normalized: false,
-      stride: 0,
-      offset: 0,
-      instanced: 0,
-      usage: gl.STATIC_DRAW
+      type: gl.FLOAT
     });
 
     const colorBuffer = new Buffer(gl, {
       data: instanceColors,
       size: 3,
-      type: gl.FLOAT,
-      normalized: false,
-      stride: 0,
-      offset: 0,
-      instanced: 1,
-      usage: gl.STATIC_DRAW
+      type: gl.FLOAT
     });
 
     for (let va = 0; va < vertexArrays.length; ++va) {
       const offsetBuffer = new Buffer(gl, {
         data: instanceOffsets,
         size: 2,
-        type: gl.FLOAT,
-        normalized: false,
-        stride: 0,
-        offset: 0,
-        instanced: 1,
-        usage: gl.STREAM_COPY
+        type: gl.FLOAT
       });
 
       const rotationBuffer = new Buffer(gl, {
         data: instanceRotations,
         size: 1,
-        type: gl.FLOAT,
-        normalized: false,
-        stride: 0,
-        offset: 0,
-        instanced: 1,
-        usage: gl.STREAM_COPY
+        type: gl.FLOAT
       });
 
       vertexArrays[va] = new VertexArray(gl, {
         buffers: {
           [COLOR_LOCATION]: {
             buffer: colorBuffer,
-            size: 3,
-            type: gl.FLOAT,
-            normalized: false,
-            stride: 0,
-            offset: 0,
             instanced: 1
```

Comment:
Nit: I think @Pessimistress is changing this to `isInstanced` in deck.gl. Some sync might be useful.